example/usps_api/server.cc

```diff
@@ -49,9 +50,9 @@ class GhostImpl final : public ghost::SfcService::Service {
 };
 
 namespace servercore {
```

Comment:
Not sure that you need this namespace. If really needed, this can be in an anonymous one.

Reply:
Done